From 44596274a63887b0c17c4ae39c24ce39ad1d8a5c Mon Sep 17 00:00:00 2001
From: Mohamed Sentissi <msentissi@anaconda.com>
Date: Wed, 18 Sep 2024 14:37:32 -0400
Subject: [PATCH] remove superfluous build requirement

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index df9f32f..ff83522 100644
--- a/setup.py
+++ b/setup.py
@@ -15,7 +15,7 @@
 from setuptools import setup
 
 
-requirements = ["requests>=2.20.0,<3.0"]
+requirements = []
 
 with open("README.md") as f:
     readme = f.read()
-- 
2.45.2

